src/modules/launcher/Wox.Plugin/Query.cs

```diff
@@ -19,32 +19,74 @@ internal Query()
         /// Initializes a new instance of the <see cref="Query"/> class.
         /// to allow unit tests for plug ins
         /// </summary>
-        public Query(string rawQuery, string search, ReadOnlyCollection<string> terms, string actionKeyword = "")
+        public Query(string query, string actionKeyword = "")
         {
-            Search = search;
-            RawQuery = rawQuery;
-            Terms = terms;
+            _query = query;
             ActionKeyword = actionKeyword;
         }
 
+        private string _rawQuery;
```

Comment:
Let's not introduce the underscore prefix for private members, since this project wasn't using it in the first place.
Also, it would be preferable to not use private variables and use get/set with the proper accessibility.

Reply:
Wox.Plugin uses underscore prefix for private members. At least I did not found any private, not static, not readonly member that does not have underscore prefix. Am I missing something?

If we use private property in order to implement public property it would be weird. We only need one get, set method to access data.